iothub/service/tests/ClientWebSocketTransportTests.cs

```diff
@@ -105,6 +105,7 @@ public async Task ClientWebSocketTransportReadWithoutConnectTest()
         // The following tests can only be run in Administrator mode
         [TestMethod]
         [Timeout(10000)] // if it is going to fail, fail fast. Otherwise it can go on for 4+ minutes. :(
+        [DoNotParallelize]
```

Comment:
This test always fails for me locally when run with the other tests in this class, so I thought we'd try this.